reproductions/Exercise/Exercise_04_06.py

```diff
@@ -1,24 +1,17 @@
 import sympy
-import numpy as np
+import numpy
+from utils import ssr_solve
 
-a11, a12, a21, a22, b21, b22, z = sympy.symbols('a11 a12 a21 a22 b21 b22 z')
+a11, a12, a21, a22, b21, b22, z = sympy.symbols(
+    'a11 a12 a21 a22 b21 b22 z'
+)
 
-A = sympy.Matrix([[a11, a12],[a21, a22]])
-B = sympy.Matrix([[1, 1],[b21, b22]])
-C = sympy.Matrix([[1, 0],[0, 1]])
-D = sympy.Matrix(sympy.Matrix(np.zeros_like(A)))
+A = sympy.Matrix([[a11, a12], [a21, a22]])
+B = sympy.Matrix([[1, 1], [b21, b22]])
+C = sympy.Matrix([[1, 0], [0, 1]])
+D = sympy.Matrix(sympy.Matrix(numpy.zeros_like(A)))
```

Comment:
I know you didn't put it there, but why have an extra Matrix call?

Reply:
I have no idea, missed it in my review. Removed double matrix and replaced redundant conversion from numpy matrix.